nautobot/extras/tables.py

```diff
@@ -295,6 +302,38 @@ class Meta(BaseTable.Meta):
         default_columns = ("pk", "created", "related_object", "user", "status", "data")
 
 
+#
+# ScheduledJobs
+#
+
+
+class ScheduledJobsTable(BaseTable):
```

Comment:
Given this is for the `ScheduledJob` model not the `ScheduledJobs` singleton table, would suggest:

```suggestion
class ScheduledJobTable(BaseTable):
```